### PR TITLE
[docs] Updated tutorials for cloud deployment

### PR DESCRIPTION
**Pull request by gradientsky**
*Description of changes:*
Updated tutorials for cloud deployment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Innixma — APPROVED** [review]
LGTM!

File: docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ def model_fn(model_dir):
     """loads model from previously saved artifact"""
     model = TabularPredictor.load(model_dir)
     globals()["column_names"] = model.feature_metadata_in.get_features()
+    predictor.persist_models()
```
Comment (Innixma) [inline docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md:40]: `predictor` is not defined.

Comment (Innixma) [inline docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md:40]: `persist_models('best')` if the user only cares about the best model.

Comment (Innixma) [reply docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md:40]: Disregard, default is best

**Review by yinweisu — APPROVED** [review]
LGTM!

**Pull request merged**